src/shapes/text.class.js

```diff
@@ -908,7 +913,7 @@
       var box = {
         width: width,
         left: 0,
-        height: charStyle.fontSize,
+        height: style.fontSize,
         kernedWidth: kernedWidth,
       };
```

Comment:
would be nice to have the deltay in the graphmebox information, even if for now we are not using it in the grapheme box context.